### ISSUE TITLE
Discrepancy in number of classes in the configuration file

### ISSUE THREAD

**Issue by faizan1041** [DevashishPrasad/CascadeTabNet#96]
I was trying to train the model on the configuration file provided here [Config/cascade_mask_rcnn_hrnetv2p_w32_20e.py](https://github.com/DevashishPrasad/CascadeTabNet/blob/master/Config/cascade_mask_rcnn_hrnetv2p_w32_20e.py). I'm failing to understand the number of classes, considering bordered, borderless and cell, it should be 3+1 classes instead of 81 which is here. Can someone provide any explanation on this?

**Comment by marcelodiaz558** [comment]
Hello, I have the same doubt, did you figured it out?

**Comment by faizan1041** [comment]
Yes, that file is a sample from somewhere else, the real number of classes should be (num_of_classes + 1)

**Comment by DB11051998** [comment]
https://github.com/open-mmlab/mmdetection/issues/6340


I guess this might clear your doubts!!
